src/TimeZoneNames.DataBuilder/DataExtractor.cs

```diff
@@ -483,15 +481,14 @@ private static TimeZoneValues GetTimeZoneValues(XContainer element)
         private void LoadDisplayNames()
         {
             using StreamReader textReader = File.OpenText(_tzresPath + "tzinfo.json");
-            using var jsonReader = new JsonTextReader(textReader);
-            var data = JObject.Load(jsonReader);
-            JToken languages = data["Languages"];
+            var json = textReader.ReadToEnd();
+            var data = JsonNode.Parse(json);
```

Comment:
We should try to avoid the intermediate string.

Reply:
Done.  I went with a stream directly to avoid the extra TextReader.